community_images/fluent-bit/ironbank/k8s_coverage.sh

```diff
@@ -12,17 +12,18 @@ SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 
 JSON_PARAMS="$1"
 
-#JSON=$(cat "$JSON_PARAMS")
+JSON=$(cat "$JSON_PARAMS")
+echo "Json params for k8s coverage = $JSON"
 NAMESPACE=$(jq -r '.namespace_name' < "$JSON_PARAMS")
 RELEASE_NAME=$(jq -r '.release_name' < "$JSON_PARAMS")
 echo "NAMESPACE: $NAMESPACE"
 echo "RELEASE_NAME: $RELEASE_NAME"
 
 sleep 10
-CONTAINER_NAME=$(kubectl get pods -n "$NAMESPACE" -l "app.kubernetes.io/instance=$RELEASE_NAME" -o jsonpath='{.items[0].metadata.name}')
-# copy over the script to the pod
-kubectl cp "${SCRIPTPATH}"/config/fluent-bit.config "${CONTAINER_NAME}":/tmp/fluent-bit.config -n "${NAMESPACE}"
-kubectl exec "${CONTAINER_NAME}" -n "${NAMESPACE}" -- /bin/bash -c "nohup /fluent-bit/bin/fluent-bit -c /tmp/fluent-bit.config" &
+
+CONTAINER_NAME="$RELEASE_NAME"
```

Comment:
Please verify if this name is correct. Refer other images.

Reply:
It is used both as "$RELEASE_NAME" and "$RELEASE_NAME"-0